velox/connectors/WriteProtocol.h

```diff
@@ -30,6 +30,11 @@ class WriterParameters {
   virtual ~WriterParameters() = default;
 };
 
+class ConnectorWriteInfo {
```

Comment:
Why do we need a separate interface? Can the necessary information be included in ConnectorInsertTableHandle?

Reply:
I'm planning to use ConnectorWriteInfo to pass the partition subdirectory (or partition name) to WriteProtocol::getWriterParameters(),  which is Hive connector specific.

We get the partition subdirectories on the fly when reading the inputs and parsing the individual records. Here is where I populate ConnectorWriteInfo in another PR, https://github.com/facebookincubator/velox/blob/feec0f6736c48297b9853d92a394aa9a656e9050/velox/connectors/hive/HiveDataSink.cpp#L212. ConnectorInsertTableHandle are passed from the coordinator and converted from the similar protocol structure, https://github.com/prestodb/presto/blob/48a7d5f477d7d1c462a0a95e5a174c63d9bb623b/presto-native-execution/presto_cpp/presto_protocol/presto_protocol.cpp#L6231-L6250.

Reply:
Thank you for explaining. Would you document this interface?